### PR TITLE
Create Calmcoder.xml

### PR DESCRIPTION
**Pull request by ctrlaltvikas**
A simple,elegant and colorful color-theme for coders.

### CODE REVIEW

**Comment by ricardograca** [comment]
Thanks! I'll check it out later today or tomorrow and either respond back with any questions/issues or merge it.

**Comment by ricardograca** [comment]
After reviewing it (sorry it took this long) I'm not a big fan of the particular shade of purple used for some keywords. This makes them quite a bit hard to read against the green background:

Calm Coder gedit theme preview

Everything else looks good to me. Do you have any intention of reviewing this?

**Comment by vikaskyadav** [comment]
Yeah. I will be more than happy. Suggest me some changes.  I will try to
implement those within this week.!

**Comment by ricardograca** [comment]
Just the purple color that's a bit hard to read. Everything else looks good.

File: Calmcoder.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+  <?xml version="1.0" encoding="UTF-8"?>
```
Comment (ricardograca) [inline Calmcoder.xml]: These two spaces you added here made the XML document invalid and caused it to not be detected by Gedit.

File: Calmcoder.xml [hunk 2]
```diff
@@ -45,20 +45,20 @@
   <style name="def:decimal" foreground="#ffcc33"/>
   <style name="def:doc-comment" foreground="#chocolate"/>
   <style name="def:doc-comment-element" foreground="#chocolate"/>
-  <style name="def:error" foreground="#f8f8f0" background="#f92672"/>
+  <style name="def:error" foreground="#f8f8f0" background="#ffcc66"/>
```
Comment (ricardograca) [inline Calmcoder.xml]: Looks good to me, good job :+1:

**Review by ricardograca — CHANGES_REQUESTED** [review]
After you fix the invalid XML issue I'll merge it. Looks good to me otherwise.

**Review by ricardograca — APPROVED** [review]

**Pull request merged**